includes/class-sensei-grading-user-quiz.php

```diff
@@ -215,6 +215,37 @@ public function display() {
 						break;
 				}
 
+				/**
+				 * Filter quiz right answer and user answer content for grading page display.
+				 * Filters the various values which are displayed in the grading admin page for each quiz question.
+				 * The expected values are type_name, right_answer, user_answer_content and grade_type
+				 *
+				 * @since 4.0.0
+				 *
+				 * @hook sensei_grading_display_quiz_question
+				 *
+				 * @param {null}   $display_values
+				 * @param {string} $type
+				 * @param {int}    $question_id
+				 *
+				 * @return {array|null} {
```

Comment:
Hmm after running `npm run build:docs` and checking out `hookdocs\index.html`, it seems that this documentation format is not compatible with the tool that generates the document. Sorry for the bad recommendation. 

Can we move the documentation of the array to the hook description and leave this as `@return {array|null}`?